src/art/AryanshGupta3/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Flag of Sierra Leone</title>
+    <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+    <canvas id="aryanshCanvas" width="500px" height="310px">
+        Flag of Sierra Leone
+    </canvas>
+    <script src="script.js"></script>
+</body>
+
```

Comment:
Please remove the extra space here (line 19 precisely). Thank you

Reply:
Changes done. 
Please review and merge. 
Thank You :)